gaia-switch.js

```diff
@@ -1,8 +1,5 @@
-(function(define){define(function(require,exports,module){
-/* jshint laxbreak:true */
-/* jshint esnext:true */
-
-'use strict';
+/* globals define */
```

Comment:
I still feel `/* globals define */` is better declare in this file since test files does not need `define`